README.md

```diff
@@ -0,0 +1,30 @@
+# Installation
+This installation assumes you have a running instance of sensu-base and for ssl that you have your ssl-certs generated by the sensu-tool.
+```
+juju deploy sensu-client
+```
+The sensu-client requires 2 config options, password, which is the vhost password for the rabbitmq-server, exposed in juju status. The other one is rabbitmq, which is the privateip:port of the rabbitmq-instance. When using ssl, 2 extra config options must be set. Assuming you are in the sensu_ssl_tool directory:
+```
+juju config sensu-client rabbitmq="private_ip:port"
+juju config sensu-client password="[rabbitmq password]"
+juju config sensu-client ssl_key="`cat client/key.pem`" ssl_cert="`cat client/cert.pem`"
+```
+Then all that remains is:
+```
+juju add-relation sensu-client your_service
```

Comment:
Also make it clear that your-service doesn't need any special relation (if that is the case).